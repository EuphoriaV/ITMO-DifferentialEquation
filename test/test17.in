3.21
((7.39 + 4.82) * x)
4.42
5.52