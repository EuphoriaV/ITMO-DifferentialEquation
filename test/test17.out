1.62E-5